vlog4j-core/src/test/java/org/semanticweb/vlog4j/core/model/ConjunctionImplTest.java

```diff
@@ -181,4 +181,19 @@ public void negativeLiteralsNoNullElements() {
 		Expressions.makeConjunction(negativeLiteralList);
 	}
 
+	@Test
+	public void testconjunctiontoString() {
```

Comment:
careful with the method names. There are various popular naming conventions (https://dzone.com/articles/7-popular-unit-test-naming), but all of them require proper camelCase.